src/Configuration/ConfigurationFactory.php

```diff
@@ -93,32 +93,16 @@ private function shouldShowDiffs(InputInterface $input): bool
         return ! SimpleParameterProvider::provideBoolParameter(Option::NO_DIFFS, false);
     }
 
-    /**
-     * @param string[] $commandLinePaths
-     * @return string[]
-     */
-    private function correctBashSpacePaths(array $commandLinePaths): array
-    {
-        // fixes bash edge-case that to merges string with space to one
-        foreach ($commandLinePaths as $commandLinePath) {
-            if (\str_contains($commandLinePath, ' ')) {
-                $commandLinePaths = explode(' ', $commandLinePath);
-            }
-        }
-
-        return $commandLinePaths;
```

Comment:
This actually incorrectly splits any spaced path to 2 strings:

```bash
/some directory/file.php
```

will result into:

```bash
/some
directory/file.php
```